modules/timepoint_list/templates/menu_timepoint_list.tpl

```diff
@@ -60,7 +60,7 @@
     <!-- </div> -->
     <!-- <div class="col-xs-4"> -->
         {$actions}
-           <button class="btn btn-primary timepoint_imaging_datasets" data-pscid="{$PSCID}">
+           <button class="btn btn-primary timepoint_imaging_datasets" onclick="self.location.href=('{$baseurl}/imaging_browser/?DCCID={$candID}')">
```

Comment:
If this is a link, why can't it be an `<a>` tag (perhaps styled as a button)? "onclick" only works with a mouse or something that can click. `<a href>` works with screen readers, tablets, keyboards, etc.

Reply:
@driusan done :)